next.config.js

```diff
@@ -29,6 +29,11 @@ const nextConfig = {
       },
     ]
   },
+  webpack: (config) => {
+    config.resolve.fallback = { fs: false, net: false, tls: false }
+    config.externals.push('pino-pretty', 'lokijs', 'encoding')
+    return config
+  },
```

Comment:
What's this for? 👀

Reply:
Fixes the console warnings while running in dev mode: https://github.com/WalletConnect/walletconnect-monorepo/issues/1908#issuecomment-1487801131

(Matches our implementation in `interfaces`: https://github.com/IndexCoop/interfaces/blob/main/apps/auctions/next.config.js#L16)